standards/imaging/1.0.0/imaging_schema.md

```diff
@@ -0,0 +1,402 @@
+
+The mapping of CryoET data and DCA data to the XMS is specified [here](https://github.com/chanzuckerberg/data-guidance/blob/main/standards/imaging/1.0.0/XMS_mapping.md).
+
+## Data description
+Administrative metadata about the source of the data, funding, relevant licenses, and restrictions on use.
+
+Cross-modality metadata: 
+Security & Compliance Metadata and Technical Metadata outlined in the [Data registration needs](https://czi.atlassian.net/wiki/spaces/SCI/pages/3896868957/Data+Registration+Needs+Metadata) are captured here.
+
+### imaging_method
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>imaging_method</td>
+    </tr>
+    <tr>
+      <th>Description</th>
+      <td>Defines the imaging method that was used to create the dataset</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Submitter MUST annotate.</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td><code>List[String]</code>. 
+         The List element MUST be a term in <a href="https://webprotege.stanford.edu/#projects/ee9f28eb-bcfe-469d-a9b4-48929f100eec/edit/Classes">the imaging methods list </a>.
+        </td>
+    </tr>
+</tbody></table>
+<br>
+
+### file_format
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>file_format</td>
+    </tr>
+    <tr>
+      <th>Description</th>
+      <td>Specify the file format</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Submitter MUST annotate.</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td><code>List[String]</code>. 
+         Examples: Zarr_V2, Zarr_V3, mrc.
+        </td>
+    </tr>
+</tbody></table>
+<br>
+
+### creation_time
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>creation_time</td>
+    </tr>
+    <tr>
+      <th>Description</th>
+      <td>The start time of data acquisition, used to uniquely identify the data.</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Submitter MUST annotate.</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td><code>datetime</code> string formatted according to 
+      <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO&nbsp;8601</a> 
+      (e.g., <code>YYYY-MM-DD</code> in UTC).</td>
+        </td>
+    </tr>
+</tbody></table>
+<br>
+
+## Subject
+Metadata describing the subject being imaged: Species, genotype, age, sex, and source.
+
+Cross-modality metadata: 
+[Cross-modality schema metadata](https://github.com/chanzuckerberg/data-guidance/tree/main/standards/cross-modality) are captured here.
+
+
+### non_biological_subject
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>non_biological_subject</td>
+    </tr>
+    <tr>
+      <th>Description</th>
+      <td>Specify the non-biological object in the data.</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>OPTIONAL.</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td><code>List[String]</code>. 
+        </td>
+    </tr>
+</tbody></table>
+<br>
+
+## Procedure
```

Comment:
I'd note that there's been conversations about similar metadata for chemical perturbations. See https://docs.google.com/document/d/1gRhhMdLaxaTilSUXwFPNxuul-fpW7ur9ztz4KPPdCP8/edit?pli=1&disco=AAABrkvLRSw and there was consensus that CELLxGENE would not capture because such information cannot be standardized/validated and would not be comparable between experiments.

Reply:
@BAevermann @jahilton ^^^ - do you wish to comment?